x-pack/legacy/plugins/canvas/public/lib/clipboard.js

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { Storage } from 'ui/storage';
+import { Storage } from '../../../../../../src/plugins/kibana_utils/public';
```

Comment:
I don't think you have to have a path like this... I seem to recall Kibana paths aliased in Webpack so you can just do `src/plugins/kibana_utils/public`...?

For example: https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/canvas/canvas_plugin_src/functions/browser/location.ts#L7

Reply:
Asked and answered: https://github.com/elastic/kibana/issues/40446

Approving for Canvas... thanks!

Reply:
@clintandrewhall I'm familiar with that issue, but as far as I know it works for TS types but not for classes \ functions. Are you aware of any changes there?